src/Features/Core/Portable/DocumentationComments/IDocumentationCommentFormattingService.cs

```diff
@@ -11,5 +12,6 @@ internal interface IDocumentationCommentFormattingService : ILanguageService
     {
         string Format(string rawXmlText, Compilation compilation = null);
         IEnumerable<TaggedText> Format(string rawXmlText, SemanticModel semanticModel, int position, SymbolDisplayFormat format = null);
+        IEnumerable<TaggedText> Format(ISymbol symbol, SemanticModel semanticModel, int position, CancellationToken cancellationToken, SymbolDisplayFormat format = null);
```

Comment:
CancellationToken should always be last.  Avoid optional parameters in internal interfaces (it's easy for callers to accidentally forget to pass a value).

Reply:
done